app/backend/src/tests/test_email.py

```diff
@@ -142,8 +147,8 @@ def mock_queue_email(sender_name, sender_email, recipient, subject, plain, html)
             assert from_date in html
             assert to_date in plain
             assert to_date in html
-            assert from_user.avatar_url not in plain
-            assert from_user.avatar_url in html
+            assert from_user.avatar_filename not in plain
+            assert from_user.avatar_filename in html
```

Comment:
Hmm, didn't you just remove avatar_filename from the User/users table?

Reply:
Yes, those tests are awful and crappy. It seems the code's not actually being run...

Reply:
Oh, really? Where's the link to the code coverage now again? Clicking the banner just brings me to the gitlab project page.

Reply:
https://backend-feature-general-media-upload--bcov.preview.coucher.org/

Reply:
Created https://github.com/Couchers-org/couchers/pull/682 to fix the url. Will make another PR to include test directory in the report.

Reply:
Created https://github.com/Couchers-org/couchers/pull/686 for including tests in coverage report.

Reply:
Created https://github.com/Couchers-org/couchers/pull/687 for fixing the tests. That PR should be merged to develop before this one.